diff --git a/cmake/Dependencies.cmake b/cmake/Dependencies.cmake
index 51a803c..9501156 100644
--- a/cmake/Dependencies.cmake
+++ b/cmake/Dependencies.cmake
@@ -118,6 +118,11 @@ if(BUILD_python)
     find_package(Boost 1.46 COMPONENTS "python-py${boost_py_version}")
     set(Boost_PYTHON_FOUND ${Boost_PYTHON-PY${boost_py_version}_FOUND})

+    if(NOT Boost_PYTHON_FOUND)
+      find_package(Boost 1.46 COMPONENTS "python${boost_py_version}")
+      set(Boost_PYTHON_FOUND ${Boost_PYTHON${boost_py_version}_FOUND})
+    endif()
+
     while(NOT "${version}" STREQUAL "" AND NOT Boost_PYTHON_FOUND)
       STRING( REGEX REPLACE "([0-9.]+).[0-9]+" "\\1" version ${version} )

@@ -125,6 +130,11 @@ if(BUILD_python)
       find_package(Boost 1.46 COMPONENTS "python-py${boost_py_version}")
       set(Boost_PYTHON_FOUND ${Boost_PYTHON-PY${boost_py_version}_FOUND})

+      if(NOT Boost_PYTHON_FOUND)
+        find_package(Boost 1.46 COMPONENTS "python${boost_py_version}")
+        set(Boost_PYTHON_FOUND ${Boost_PYTHON${boost_py_version}_FOUND})
+      endif()
+
       STRING( REGEX MATCHALL "([0-9.]+).[0-9]+" has_more_version ${version} )
       if("${has_more_version}" STREQUAL "")
         break()
